go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 require (
 	github.com/go-logr/logr v1.4.1
 	github.com/google/safetext v0.0.0-20230106111101-7156a760e523
-	github.com/gophercloud/gophercloud/v2 v2.0.0-beta.3.0.20240416104816-9aef3836d310
+	github.com/gophercloud/gophercloud/v2 v2.0.0-beta.4.0.20240422125343-8b1eebe1fa46
```

Comment:
We took 0.3 + commit to include a very specific fix. Is the HTTP change you refer to on top of beta.4 the same way or could we just take beta.4?

Reply:
I have selfish reasons here: the most recent commit is needed for https://github.com/metal3-io/baremetal-operator/pull/1689.

Reply:
:+1:  Fair enough, was just making sure we don't do it for no reason.